features/query/timeout_due_to_no_active_workers/feature.go

```diff
@@ -0,0 +1,47 @@
+package timeout_due_to_no_active_workers
+
+import (
+	"context"
+	"go.temporal.io/api/serviceerror"
+	"go.temporal.io/sdk-features/harness/go/harness"
+	"go.temporal.io/sdk/client"
+	"go.temporal.io/sdk/workflow"
+	"time"
+)
+
+var Feature = harness.Feature{
+	Workflows: Workflow,
+	CheckResult: func(ctx context.Context, r *harness.Runner, run client.WorkflowRun) error {
+		// Shut off the worker
+		r.Worker.Stop()
```

Comment:
I think it'd be clearer to have a helper to stop the worker (e.g. `r.StopWorker()`) that would properly mark it stopped by setting the worker to nil and not try to stop at the end if it was already stopped (i.e. nil). Also `RestartWorker` (which I propose changing to `StartWorker`) will fail on a non-nil worker (i.e. already running).